init.sh

```diff
@@ -1,4 +1,7 @@
-KEY="mykey"
+#!/bin/bash
+KEYS[0]="mykey"
```

Comment:
I decided to change these to `mykey`, `mykey1`, `mykey2` instead of `dev0`, `dev1`, ... because of better readability for the docs? 
If we want to keep all of the documentation directly copy-and-pasteable for the init-config, we would need to change everything to e.g. `evmosd keys show dev0` instead of `evmosd keys show mykey` etc.

I would prefer this option, wdyt? @danburck @fedekunze

Reply:
@MalteHerrmann both is fine for me. I'm biased as I've used mykey a lot in the past.

Reply:
Changed to `dev0`, ... according to Fede's preference